example/_config.yml

```diff
@@ -8,6 +8,9 @@ description: > # this means to ignore newlines until "baseurl:"
 baseurl: "/minima"
 twitter_username: jekyllrb
 github_username:  jekyll
+# if you want to use disqus as comment system, be aware that it only works on production
+disqus:
+  shortname: your_disqus_shortname
```

Comment:
Should this be commented out by default, such that we don't have a bunch of sites, which by default, have a broken comment system? Also, perhaps we should provide details on how to sign up?